src/entities/protocols/convertWETH.ts

```diff
@@ -0,0 +1,33 @@
+import invariant from 'tiny-invariant'
+import { BigNumberish } from 'ethers'
+import { RoutePlanner, CommandType } from '../../utils/routerCommands'
+import { Permit2Permit } from '../../utils/permit2'
+import { Command, RouterTradeType, TradeConfig } from '../Command'
+import { encodePermit } from '../../utils/permit2'
+import { ROUTER_AS_RECIPIENT, UNIVERSAL_ROUTER_ADDRESS, WETH_ADDRESS } from '../../utils/constants'
+
+export class ConvertWETH implements Command {
```

Comment:
Maybe this should be called `UnwrapWETH` or something else? To me `ConvertWETH` was going to be something that could handle both ETH->WETH and WETH-ETH. But this class only handles unwrapping